compiler/crates/relay-bin/src/main.rs

```diff
@@ -57,6 +74,25 @@ struct Opt {
     validate: bool,
 }
 
+#[derive(Parser)]
+#[clap(about = "Run the LSP server", rename_all = "camel_case")]
+struct LspCommand {
+    /// Run the LSP using this config file. If not provided, searches for a config in
+    /// package.json under the `relay` key or `relay.config.json` files among other up
+    /// from the current working directory.
+    config: Option<PathBuf>,
+
+    /// Verbosity level
+    #[clap(long, arg_enum, default_value = "verbose")]
```

Comment:
Does this show up in the "Output" panel in VSCode? If not, I think we should use a lower level here, like `info` or `quiet`, since 99% of the time nobody would see it, and it would just burn cycles.

Reply:
Great callout!